src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,12 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService usr = new UserService();
+        User usrdata = new User(email, password);
+        if (usr.findByEmail(email) != null) {
+            usrdata = usr.findByEmail(email);
```

Comment:
let's save `userService.findByEmail(email)` into a local variable before `if` check

please use proper variable names
`usr`, `usrdata` are not good names:

- we should not shorten words
- we should use camelCase for variable names